lib/git-manager.js

```diff
@@ -326,6 +329,27 @@ function params(apiParams) {
 }
 
 /**
+ * Returns a function that uses the search parameters to add the "repository"
+ * property to all the results. When we ask for a list of open issues from the
+ * API for a particular repo, those results don't have references to the repo
+ * we asked about, so we have to inject them to normalize the structure of the
+ * object.
```

Comment:
Its a four line run-on?
> so we have to inject them
Inject what?

I get the intent of the comment, but it's really opaque.